src/tracepoint_config.cc

```diff
@@ -23,6 +23,9 @@ int64_t tracepoint_get_id(std::string_view global_name,
     fs_path = "/sys/kernel/tracing/events";
   } else if (stat("/sys/kernel/debug/tracing/events", &sb) == 0) {
     fs_path = "/sys/kernel/debug/tracing/events";
+  }
+  if (stat("/proc/1/root/sys/kernel/debug/tracing/events", &sb) == 0) {
+    fs_path = "/proc/1/root/sys/kernel/debug/tracing/events";
```

Comment:
```suggestion
  } else if (stat("/proc/1/root/sys/kernel/debug/tracing/events", &sb) == 0) {
    fs_path = "/proc/1/root/sys/kernel/debug/tracing/events";
```

Reply:
Precise binding order TBD.  I weakly suspect that preferring "official" tracefs or debugfs is preferable, but either way should probably not over-write a previous selection at the end.